docs/content/users/quickstart.md

```diff
@@ -607,6 +607,65 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch /cp
     ```
 
+## Sulu
+
+```bash
+mkdir my-sulu-site && cd my-sulu-site
+ddev config --project-type=php --docroot=public
+ddev start
+ddev composer create sulu/skeleton
+```
+
+!!!tip "Multilingual"
+    To set up the administration interface for example in French alongside the default English, add the following snippet on top of `config/packages/sulu_admin.yaml` ([all available localizations](https://sulu.crowdin.com/sulusulu)):
+
+    ```bash
+    sulu_core:
+    locales:
+        en: English
+        fr: Francais
+    translations:
+        - en
+        - fr
+    ```
+
+    `ddev exec bin/console sulu:admin:download-language` downloads any set language. The only exceptions are English and German which are shipping with Sulu.
+
+The content management part of Sulu is built upon webspaces. Each webspace configures a content tree. Each content tree may contain translations for different locales. Create your default webspace configuration `mv config/webspaces/example.xml config/webspaces/my-sulu-site.xml` and adjust the values for `<name>` and `<key>` so that they are matching your project:
+
+```bash
+<?xml version="1.0" encoding="utf-8"?>
+<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
+    <!-- See: http://docs.sulu.io/en/latest/book/webspaces.html how to configure your webspace-->
+
+    <name>My Sulu CMS</name>
+    <key>my-sulu-cms</key>
+```
+
+!!!warning "Caution"
+    Changing the `<key>` for a webspace later on causes problems. It is recommended to decide on the value for the key before the database is build in the next step.
+
+The information for the database connection is set in the environment variable `DATABASE:URL`. For development, clone the `.env` file by `cp .env .env.local` and change the environment variable to:
```

Comment:
This line is for all Symfony based applications not correct the `.env`. You never do `cp .env .env.local` in Symfony based applicatoins as the `.env` is always loaded. You only write into the `.env.local` the things you want to change so in this case DATABASE_URL is only required to be added the `.env.local` normally already exists as the composer create project already set `APP_ENV=dev` for quick development start by Sulu.

```suggestion
The information for the database connection is set in the environment variable `DATABASE_URL`. The installation should have already created a `.env.local` file which allows you overwrite the `DATABASE_URL` if no `.env.local` exists create it and define also the `APP_ENV` variable:
```

Reply:
at first thank you for the review! and uh good point, cloning is not the right way to handle things here since `.env.local` is just overriding, that was my fault. 
but one question when i am applying the steps in the current draft `.env.local` is not created automatically by `ddev composer create sulu/skeleton` only `.env`, `.env.stage`, and `.env.test` is being created. To stay directly actionable and under the assumption that the `.env.local` file is not being created yet, how about using something like that instead of your second sentence? 

```
For development, override the `APP_ENV` and `DATABASE_URL` variables in the `.env.local` file with:

echo -n "APP_ENV=local\nDATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7" >> .env.local
```

Reply:
@rpkoller does ddev skip the composer create project scripts? Its create by the basic composer root install part:

https://github.com/sulu/skeleton/blob/6f9ff17f8862c482a5a4155f916b22b7db216112/composer.json#L120

That part should not be skipped, I recommend that for any framework / system:


![sulu-create-project](https://github.com/ddev/ddev/assets/1698337/a6081474-cbc8-4947-81af-4debba75af32)

Reply:
oh that is interesting. it looks like that some of the project scripts are skipped and not executed when running  `ddev composer create sulu/skeleton`. here is the output of the first two initial steps: https://gist.github.com/rpkoller/f4be5ff370afd60fe972b70275deddbd . those steps after the project is being create are missing. for example the app secret in the .env file is also not being replaced. i guess that bug in ddev first has to be fixed,  cuz it makes a few of the steps in the draft obsolete.

Reply:
There is some problem with Mutagen sync in DDEV v1.23.1, `.env.local` is not created on `ddev composer create sulu/skeleton`

But @rfay recently made some changes for Mutagen, I tried `ddev composer create sulu/skeleton` with DDEV HEAD, and this problem is fixed, `.env.local` will be there is DDEV v1.23.2.

Reply:
i am on the latest version of HEAD (ddev version v1.23.0-84-g213f46972 - just tested if there was another commit). so the output in https://gist.github.com/rpkoller/f4be5ff370afd60fe972b70275deddbd shows that those scripts are  still not executed unfortunately. and i am also unsure how mutagen could interfere with the scripts not being executed?

Reply:
The problem is that somebody add `--no-scripts` see output line 24:

```
Executing Composer command: [composer create-project sulu/skeleton --no-plugins --no-scripts --no-install /tmp/UPPlPA]
```

Reply:
one script is executed at a later point `composer run-script post-create-project-cmd` but not all scripts in general https://gist.github.com/rpkoller/f4be5ff370afd60fe972b70275deddbd#file-2-ddev-composer-create-txt-L543 aside line 536-540

Reply:
The `post-root-package-install` should be executed before `Executing Composer command: [composer install]` (L54)

Reply:
We have `composer run-script post-root-package-install` script instruction (before `composer install`), but I tested this on Linux w/ and w/o Mutagen.

I'll definitely have to check it out again on macOS.

Reply:
the setup the gist is created with is macos 14.5 running on lima with VZ

Reply:
I reproduced it with macOS, and also with Linux (I don't know why I couldn't the first time, maybe I mixed the binaries)

I opened this:

- #6281